docs/Getting-Started.md

```diff
@@ -0,0 +1,178 @@
+To get started hacking with Tux, let's see what some parts of a to-do component might look like inside a To-Do Application built with Tux.
+
+To begin with we'll require only the modules we need to get our component up and running:
+
+```javascript
+    var React = require('tux/React');
+
+```
+
+Let's start by creating the top level Todo component using Tux's `createOwnerClass`. The Owner class component handles the state for our Todo application. In `Tux` you are able to easily connect your Owner components to their stores using the `connectOwnerToStore` method. As the top-level Todo component it is also be responsible for exposing its methods and properties its children components. This is accomplished by using the `registerOwnerProps` method, which allows you to specify which properties and methods you want to expose to children components. Note these properties and methods should be static, if you need to pass down mutable properties, use the standard `React` practices to flow down `props`.
```

Comment:
Change second sentence to ...it is also responsible...

Change third sentence to ...which allows you to designate static properties and methods you want to automatically expose child components on the key of `nearestOwnerProps`.